Array_Rotation/Array_Rotation.java

```diff
@@ -21,18 +30,32 @@ private static void Rotate(int arr[], int rotations, int n)
         for (int i = 0; i < n; i++) 
             System.out.print(arr[i] + " ");    
     } 
+    private static void RotateDriverRight(int arr[], int rotations, int n) 
+    { 
+        // call for rotation one by one
+        for (int i = 0; i < rotations; i++) 
+            rightRotate(arr, n); 
+
+        // print the array
+        for (int i = 0; i < n; i++) 
+            System.out.print(arr[i] + " ");    
+    } 
 
     public static void main(String args[]) 
     { 
         // Sample input
-        int arr[] = {1, 2, 3, 4, 5}; 
-        int n = arr.length; 
-        Rotate(arr, 2, n);
+        int arr1[] = {1, 2, 3, 4, 5}; 
+        int arr2[] = {1, 2, 3, 4, 5};
+        int n = arr1.length; 
+        int m = arr2.length;
+        RotateDriverLeft(arr1, 2, n);
+        System.out.println();
+        RotateDriverRight(arr2, 2, m);
```

Comment:
Rename the functions to RotateArrayLeft & ..Right to be more clear

Reply:
I have renamed the functions as per your suggestions